cluster/examples/kubernetes/ceph/cluster-on-local-pvc.yaml

```
@@ -231,6 +231,8 @@ spec:
               volumeMode: Block
               accessModes:
                 - ReadWriteOnce
+    # when onlyApplySpecificOSDPlacement false, will merge both placement.All() and storageClassDeviceSets.Placement
```

I would suggest something like 
```suggestion
    # onlyApplySpecificOSDPlacement  will only use `placement` or  `preparePlacement` under `storageClassDeviceSets` when set to true.  In case of false,  `placement.All` will be merged with `placement` or  `preparePlacement` under `storageClassDeviceSets`
```